ipi/utils/io/backends/io_pdb.py

```diff
@@ -116,6 +116,12 @@ def read_pdb(filedesc, **kwargs):
         # skip the comment field
         comment = copy.copy(header)
         header = filedesc.readline()
+    if 'positions{' not in comment:
+        comment = comment.strip()
+        comment += ' positions{angstrom}\n'
```

Comment:
I would still capitalize "Angstrom", at least in the code, even if it does not influence functionality.